tests/test_visitors/test_ast/test_operators/test_walrus.py

```diff
@@ -18,6 +18,13 @@
     if y > 2
 ]
 """
+correct_walrus_comprehension = """
+some = [
+    x + y
```

Comment:
Please, also test that you can use `:=` in the expression, like `{(key := compute_key(i)): values[key] for i in range(10)}`